provider/cmd/pulumi-resource-random/schema.json

```diff
@@ -98,22 +64,19 @@
             ],
             "inputProperties": {
                 "byteLength": {
-                    "type": "integer",
-                    "description": "The number of random bytes to produce. The\nminimum value is 1, which produces eight bits of randomness.\n",
-                    "willReplaceOnChanges": true
+                    "type": "number",
+                    "description": "The number of random bytes to produce. The minimum value is 1, which produces eight bits of randomness.\n"
                 },
                 "keepers": {
                     "type": "object",
                     "additionalProperties": {
-                        "$ref": "pulumi.json#/Any"
+                        "type": "string"
                     },
-                    "description": "Arbitrary map of values that, when changed, will\ntrigger a new id to be generated. See\nthe main provider documentation for more information.\n",
-                    "willReplaceOnChanges": true
```

Comment:
We need `willReplaceOnChanges` support too - otherwise the docs will not specify the resource will cause replacement. I assume the underlying terraform resource will trigger replacement on change at the moment (planModifier has `RequiresReplaceIfValuesNotNull`)?

Reply:
Yes it works now at runtime but we don't detect in the schema.